docusaurus/docs/React/Customization/_category_.json

```diff
@@ -1,4 +0,0 @@
-{
-  "label": "General Customization",
```

Comment:
was this supposed to be deleted? something to do with the capital C?

Reply:
Yeah, I changed the directory to lower case. So this one is technically deleted